modules/claims_api/spec/requests/v1/forms/526_spec.rb

```diff
@@ -601,6 +602,21 @@
           }
         end
 
+        context "When an activeDutyBeginDate is before a Veteran's 13th birthday" do
+          it 'raise an error' do
+            mock_acg(scopes) do |auth_header|
+              VCR.use_cassette('claims_api/bgs/claims/claims') do
+                VCR.use_cassette('claims_api/brd/countries') do
+                  headers['X-VA-Birth-Date'] = '1986-05-06T00:00:00+00:00'
```

Comment:
How does this test work? Would it be better to also include the  `activeDutyStartDate` here, so that the test is self-contained?